### PR TITLE
Version 0.1

### PR DESCRIPTION
**Pull request by NotMarx**
Full changelogs will be displayed on the releases tag.

### CODE REVIEW

**Review by Veilorth — CHANGES_REQUESTED** [review]
`embed` was deprecated as you mentioned previously. I prefer to have this changed rather than deprecating it away.

File: lib/Errors/ReloadedError.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const Messages = require("./Messages");
+
+/**
+ * A generic error from the library.
+ * @extends Error
+ * @private
+ */
+class ReloadedError extends Error {
+    constructor(code, type = "LIBRARY") {
```
Comment (Veilorth) [inline lib/Errors/ReloadedError.js:9]:
```suggestion
    constructor(code, type = "RELOADED") {
```
`RELOADED` will sound better.

**Review by Veilorth — CHANGES_REQUESTED** [review]
Last one here. It may have other things that need to be changed but I won't bother looking at it now.

File: lib/Errors/ReloadedError.js (hunk 1 above)
Comment (NotMarx) [reply lib/Errors/ReloadedError.js]: I don't see any difference but ok

**Review by Veilorth — APPROVED** [review]

**Pull request merged**